core/src/main/java/com/datastrato/gravitino/storage/relational/service/RoleMetaService.java

```diff
@@ -5,22 +5,38 @@
 package com.datastrato.gravitino.storage.relational.service;
 
 import com.datastrato.gravitino.Entity;
+import com.datastrato.gravitino.MetadataObject;
 import com.datastrato.gravitino.NameIdentifier;
 import com.datastrato.gravitino.authorization.AuthorizationUtils;
+import com.datastrato.gravitino.authorization.SecurableObject;
 import com.datastrato.gravitino.exceptions.NoSuchEntityException;
 import com.datastrato.gravitino.meta.RoleEntity;
 import com.datastrato.gravitino.storage.relational.mapper.GroupRoleRelMapper;
 import com.datastrato.gravitino.storage.relational.mapper.RoleMetaMapper;
+import com.datastrato.gravitino.storage.relational.mapper.SecurableObjectMapper;
 import com.datastrato.gravitino.storage.relational.mapper.UserRoleRelMapper;
+import com.datastrato.gravitino.storage.relational.po.CatalogPO;
+import com.datastrato.gravitino.storage.relational.po.FilesetPO;
+import com.datastrato.gravitino.storage.relational.po.MetalakePO;
 import com.datastrato.gravitino.storage.relational.po.RolePO;
+import com.datastrato.gravitino.storage.relational.po.SchemaPO;
+import com.datastrato.gravitino.storage.relational.po.SecurableObjectPO;
+import com.datastrato.gravitino.storage.relational.po.TablePO;
+import com.datastrato.gravitino.storage.relational.po.TopicPO;
 import com.datastrato.gravitino.storage.relational.utils.ExceptionUtils;
 import com.datastrato.gravitino.storage.relational.utils.POConverters;
 import com.datastrato.gravitino.storage.relational.utils.SessionUtils;
+import com.google.common.base.Splitter;
+import com.google.common.collect.Lists;
 import java.util.List;
 
 /** The service class for role metadata. It provides the basic database operations for role. */
 public class RoleMetaService {
   private static final RoleMetaService INSTANCE = new RoleMetaService();
+  private static final Splitter DOT_SPLITTER = Splitter.on('.');
+  private static final String ROOT = "ROOT";
+  private static final String ALL_METALAKES = "*";
+  private static final long ALL_METALAKES_ENTITY_ID = 0;
```

Comment:
I believe you also define this in other place, you should define them in a consoliated place, besides, this entity id should be defined in the Entity definition there, not here.

Can you please carefully think of how to better organize the code, not just simply achieve it.